addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -36,6 +36,7 @@
     .btn {
         border: none;
         border-radius: 0;
+        padding: 0.375rem 0.75rem;
```

Comment:
Any idea why the mixin o-w-preserve-btn is not enough in this case?

Reply:
@qsm-odoo You mean replace this line by `@include o-w-preserve-btn;` ?
Or you mean that there is a o-w-preserve-btn mixin somewhere that should have an impact on this button

Reply:
All of that. Are those btn already impacted by the mixin? If yes, should not that mixin add those paddings? If no, is not there a missing call to that mixin somewhere?

Reply:
@qsm-odoo actually the mixin is used in the `#oe_snippets` element. (and not really usefull cause there is not .btn inside it)
the top actions buttons are not inside this `#oe_snippets` element so I should add the mixin at an other place. But Im wondering if the best is not to leave my code like it.. Because even if I add the mixin, I have to overidde some stuff like the colors etc. .. I mean if I just need the mixin for the padding and override some other stuffs (and also take the priority with biggest selector) It's seems more complicated than just add a line for the padding